contracts/swap/swap.go

```diff
@@ -96,6 +80,24 @@ type simpleContract struct {
 	address  common.Address
 }
 
+// Deploy deploys an instance of the underlying contract and returns its address
+func Deploy(auth *bind.TransactOpts, backend bind.ContractBackend, owner common.Address, harddepositTimeout time.Duration) (common.Address, *types.Transaction, *contract.SimpleSwap, error) {
+	addr, tx, i, err := contract.DeploySimpleSwap(auth, backend, owner, big.NewInt(int64(harddepositTimeout)))
```

Comment:
please name this new variable something more explicit than `i`

Reply:
Renamed to `simpleSwap`